bin/dev.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+systemfd --no-pid -s http::0.0.0.0:7878 -- cargo watch -x run
```

Comment:
In the documentation of actix, they say that using cargo watch should be enough for the development workflow, what do you think? 

In the historical notes they talk about systemfd and listenfd: https://actix.rs/docs/autoreload/

Reply:
Yes, I did the same setup as for a project I have with Actix 2, so yes this is not required anymore!

Thanks for catching this!